docs/articles/nunit/writing-tests/assertions/classic-assertions/Assert.AreEqual.md

```diff
@@ -29,11 +27,7 @@ as equal.
 
 Special values are handled so that the following Asserts succeed:
 
-```csharp
-ClassicAssert.AreEqual(double.PositiveInfinity, double.PositiveInfinity);
-ClassicAssert.AreEqual(double.NegativeInfinity, double.NegativeInfinity);
-ClassicAssert.AreEqual(double.NaN, double.NaN);
-```
+[!code-csharp[AreEqualExamples](~/snippets/Snippets.NUnit/ClassicAssertExamples.cs#AreEqualExamples)]
```

Comment:
@SeanKilleen The same examples are shown on both line 20 and line 30.
The text might need adjusting